### ISSUE TITLE
Seeing random instances of Analytic Recommendations not being displayed in OSIO space dashboard - but recommendations are included in detailed report

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#642]
Steps to recreate - this has been seen only once today:
* Create new space
* Create new quickstart project - Vert.X basic
* At completion of pipeline build, observe no recommendations:

![screenshot 7](https://user-images.githubusercontent.com/642621/29573484-185d1582-872d-11e7-9250-96b4879e38d4.png)
 
In contrast, creating another project from the same quickstart did generate the recommendations:
![screenshot 9](https://user-images.githubusercontent.com/642621/29573725-b9cefed0-872d-11e7-9b22-fffecbe08d64.png)

Both projects showed the same information in the detailed reports:

![screenshot 8](https://user-images.githubusercontent.com/642621/29573766-ce51c31a-872d-11e7-95ef-a378a481f142.png)

**Comment by ldimaggi** [comment]
Is there a log that can be viewed to debug this issue? Thx!

**Comment by GeorgeActon** [comment]
@samuzzal-choudhury Can you poit Len in the right direction

**Comment by jyasveer** [comment]
@ldimaggi @GeorgeActon @samuzzal-choudhury  this is a known issue we are facing in the Analytics dashboard widget. There is no error anywhere. Just that the dashboard does not auto refreshes(there is a issue). If we refresh the page we do get to see the recommendations that are available. There is also a bug raised that needs to be fixed https://github.com/fabric8-ui/fabric8-ui/issues/1898

**Comment by joshuawilson** [comment]
Let's track this here. Is this still an issue?

**Comment by ldimaggi** [comment]
I think that we should keep this open - is there any way that we can push information to the user to inform him/her that a new analytic report is ready to be reviewed?

**Comment by joshuawilson** [comment]
@krishnapaparaju @samuzzal-choudhury @jyasveer what do you think? If you have the API, then updating the UI should be easy enough.

**Comment by samuzzal-choudhury** [comment]
@joshuawilson @ldimaggi The existing RESTful API setup requires the UI to poll intermittently to get the recommendations. A user will be able to get recommendations when those are ready when we introduce a web socket API.

This is going to require changes in both backend and frontend and IMHO it will be meaningful to keep this open as a feature request rather than a bug. CC: @krishnapaparaju @GeorgeActon @jyasveer

**Comment by qodfathr** [comment]
By convention, the Analysis screen is dynamic. Note that all other data on that screen updates in (near) real-time. The stack report widget should adopt the same convention, so I recommend changing this to type/bug. Since others have been arguing against that, I won't do so without first hearing new arguments as to why this should remain type/feature.

**Comment by ldimaggi** [comment]
I'd vote for changing this to a bug - from a user's perspective all that they know and see is that the report is empty.

**Comment by miteshvp** [comment]
@jyasveer - I guess this has been handled. Can you point me to the PR? If not, any ETA on this?

**Comment by jyasveer** [comment]
@miteshvp We had a Stack Analysis Report revamp. This has been handled where the dashboard would show the recommendations for the latest pipeline build. We can close this issue
